### PR TITLE
Fix local variable is assigned to but never used

### PR DESCRIPTION
**Pull request by cclauss**
% `flake8 --select=F841 --statistics`
```
./gamestonk_terminal/technical_analysis/volume.py:132:13: F841 local variable 'axVolume' is assigned to but never used
./gamestonk_terminal/technical_analysis/volume.py:241:13: F841 local variable 'axVolume' is assigned to but never used
./gamestonk_terminal/technical_analysis/overlap.py:251:13: F841 local variable 'axVolume' is assigned to but never used
./gamestonk_terminal/sentiment/twitter_api.py:169:13: F841 local variable 'n_pct' is assigned to but never used
```
We are assigning values to `n_pct` and `axVolume` but we are never using the results.  __Could these lines of code be safely eliminated?__

If there are desirable side effects from calling `axPrice.twinx()` then both flake8 and pylint should be satisfied with
`_ = axPrice.twinx()` instead of assigning to _named_ variable `axVolume`.

### CODE REVIEW

**Review by DidierRLopes — APPROVED** [review]
Yea, the twin figure was used so that we could use the right side of the plot to indicate volume.

We don't use that axis per se, but the fact that it is called makes so that when we call plt.xxx() it will be applied to it.

The solution that you pointed (_ = axPrice.twinx()) should work perfectly

File: gamestonk_terminal/sentiment/twitter_api.py [hunk 1]
```diff
@@ -163,10 +163,10 @@ def inference(l_args, s_ticker):
         if df_tweets["sentiment_estimation"].values[-1] > 0:
             n_pos = df_tweets[df_tweets["prob_sen"] > 0]["prob_sen"].sum()
             # pylint: disable=unused-variable
-            n_pct = round(100 * n_pos / df_tweets["probability"].sum())
+            n_pct = round(100 * n_pos / df_tweets["probability"].sum())  # noqa: F841
         else:
             n_neg = abs(df_tweets[df_tweets["prob_sen"] < 0]["prob_sen"].sum())
-            n_pct = round(100 * n_neg / df_tweets["probability"].sum())
+            n_pct = round(100 * n_neg / df_tweets["probability"].sum())  # noqa: F841
```
[the remark this answers is not in the document]
Comment (cclauss) [reply gamestonk_terminal/sentiment/twitter_api.py:169]: These checks are are a good way to find things that we have overlooked.

**Comment by cclauss** [comment]
Using `_` instead of `axVolume` allowed us to placate flake8 and pylint.  Do the plots come out OK with that change?

**Comment by DidierRLopes** [comment]
> Using `_` instead of `axVolume` allowed us to placate flake8 and pylint. Do the plots come out OK with that change?

Yep, they do. I'm refactoring TA folder and already doing this.

I'm also improving plotting by not blocking the matplotlib figure based on #23 . I'll leave the improvement of hint typing for another weekend though :)

**Comment by cclauss** [comment]
> I'll leave the improvement of hint typing for another weekend though :)

The general recommendation is _gradual typing_ which means do it over several PRs.  I will try to get mypy turned on so we have guardrails in place for typing.

**Pull request merged**